### ISSUE TITLE
Winston's defaultLogger property should be renamed to default

### ISSUE THREAD

**Issue by westy92** [DefinitelyTyped/DefinitelyTyped#12546]
- [x] I tried using the latest `winston/winston.d.ts` file in this repo and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] I want to talk about `winston/winston.d.ts`.
  - The authors of that type definition are cc/ @vvakame @CodeAnimal @MugeSo @horiuchi @spazard1 @jcristovao @nimish @DanielRosenwasser @Stephn-R @CaselIT @Bartvds @AyaMorisawa @awarecan @trainerbill

There is no property `defaultLogger` for winston.  Instead, there is `default` (https://github.com/winstonjs/winston/blob/master/lib/winston.js#L157)

I believe `defaultLogger` was chosen because `default` is a keyword.  However, this forces me to use `winston['default']` to access the property. Also, trying to access `winston.defaultLogger` will not yield a defined value.

**Comment by DanielRosenwasser** [comment]
For context, [here's a link to the current state of the 2.0 type declarations](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fc0c17b5efcddb5abf5539d610cfaa5f530e5f1d/winston/index.d.ts).

This file have to be rewritten to use an interface to declare Winston's exports, along with a variable and a namespace that has no value side:

```ts
declare var winston: Winston;
export = winston;

// Exported values go here.
interface Winston {
    transports: Transports;
    Transport: TransportStatic;
    Logger: LoggerStatic;
    Container: ContainerStatic;
    loggers: ContainerInstance;
    default: LoggerInstance;
    // ...
}

// Exported types go here.
declare namespace winston {
    export interface ExceptionProcessInfo {
        pid: number;
        uid?: number;
        gid?: number;
        cwd: string;
        execPath: string;
        version: string;
        argv: string;
        memoryUsage: NodeJS.MemoryUsage;
    }
}
```

**Comment by DABH** [comment]
@westy92 @DanielRosenwasser Should this issue go ahead and be closed since #12551 was merged in?
